addons/mail/static/src/core/common/chat_window.js

```diff
@@ -91,6 +91,11 @@ export class ChatWindow extends Component {
 
     onKeydown(ev) {
         const chatWindow = toRaw(this.props.chatWindow);
+        if (this.threadActions.activeAction) {
+            this.threadActions.activeAction.close();
+            ev.stopPropagation();
+            return;
+        }
```

Comment:
Should limit to "Escape", otherwise some features do not work in these panels, e.g. "Invite People" > search + filter through input closes the "Invite People" panel